cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -96,14 +94,20 @@ $ odo config set --env DOTNET_STARTUP_PROJECT=<path to your project file>
 [id="odo-technology-preview_{context}"]
 == Technology Preview features `{odo-title}`
 
-* `odo debug` is a feature that allows users to attach a local debugger to a component running in the Pod. To learn more, see xref:../../cli_reference/developer_cli_odo/debugging-applications-in-odo.adoc#debugging-an-application_debugging-applications-in-odo[Debugging applications in odo]
+* `odo debug` is a feature that allows users to attach a local debugger to a component running in the Pod. 
+To learn more, see xref:../../cli_reference/developer_cli_odo/debugging-applications-in-odo.html#debugging-an-application_debugging-applications-in-odo[Debugging applications in odo]
 
 :FeatureName: odo debug
 include::modules/technology-preview.adoc[leveloffset=+1]
 
-* Devfile support. You can now create and deploy your applications using the devfiles with `{odo-title}`. A devfile is a file that defines the development environment: environment variables, image, and so on. To access this feature, you must enable experimental mode with `odo preference set experimental true`.
+* Devfile support. With {odo-title}, you can create and deploy your applications using the devfiles. To learn more, see xref:../../cli_reference/developer_cli_odo/creating-applications-by-using-devfiles.adoc#creating-a-java-application-by-using-a-devfile[Creating applications by using devfiles]. To access this feature, you must enable experimental mode with `odo preference set experimental true`.
 +
-To see the list of currently supported devfile components, run `odo catalog list components`
+To see the list of currently supported devfile components, run `odo catalog list components`.
 
 :FeatureName: Devfile support
 include::modules/technology-preview.adoc[leveloffset=+1]
+
+* Operators support. You can now create services from Operators with `{odo-title}`. To learn more, see xref:../../cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-operators.adoc[Creating instances of services managed by Operators] To access this feature, you must enable experimental mode with `odo preference set experimental true`.
```

Comment:
Earlier, you capitalized "experimental mode" so I'd capitalize here as well for consistency.